src/generator/WDL/entities/WorkflowGenerator.js

```diff
@@ -8,7 +8,14 @@ import { buildDeclarations } from '../utils/utils';
 
 export default class WorkflowGenerator {
   constructor(wfStep, settings) {
+    this.wfBodyElementGenerators = {
+      call: this.genCall,
+      scatter: this.genScatter,
+      whileloop: this.genWhile,
+      if: this.genIf,
+    };
```

Comment:
What is the justification for the `whileloop` name? You've used `if` and `scatter` but not `while`. Consider changing the name.